src/sentry/uptime/autodetect/notifications.py

```diff
@@ -0,0 +1,120 @@
+from __future__ import annotations
+
+import logging
+
+from sentry import features
+from sentry.models.organization import Organization
+from sentry.models.project import Project
+from sentry.tasks.base import instrumented_task
+from sentry.taskworker.namespaces import uptime_tasks
+from sentry.uptime.models import get_uptime_subscription
+from sentry.uptime.types import UptimeMonitorMode
+from sentry.utils.email import MessageBuilder
+from sentry.utils.email.manager import get_email_addresses
+from sentry.utils.http import absolute_uri
+from sentry.workflow_engine.models.detector import Detector
+
+logger = logging.getLogger(__name__)
+
+
+def get_user_ids_to_notify_from_project(project: Project):
+    """
+    Get all user IDs who should be notified about auto-detected monitors in a project.
+    This returns all members of teams that are part of the project.
+    """
+    return project.member_set.values_list("user_id", flat=True)
```

Comment:
Do we definitely want to spam all members?  Or should we limit it to specific roles?